### DISCUSSION TITLE
Updating ALS to new version

### DISCUSSION THREAD

**Discussion by AmirAce** [Sixze/ALS-Refactored#369] [Q&A]
Hello Folks!

Ive been Dabbling with ALS refactored and love it! very well organized, and this disscusion forum is where I get all my answers from! now one thing I coulndt find a subject on is if there is a way to update als to a newer version without having to manually do all the changes have done before?

Thank you!

**Comment by 4re** [comment]
As far as I know, if you touched any file inside the plugin itself you are screwed.

My advice is to never directly touch anything in there and use child classes to extend the ones from the plugin (when possible).

**Reply by AmirAce** [reply]
oh I see, but is there a way to update als refactored if say I havent modified any of the plugin files?

**Reply by AmirAce** [reply]
Also I do want to mention this is for when the plugin comes out for 5.3, so I can update my engine to 5.3 and have als for ue5.3 as well

**Reply by 4re** [reply]
Then there should be any unsolvable problem, make a backup if you are concerned and just replace Unreal and the plugin with the new one.

I'm already using the main branch with unreal 5.3, and so far so good.

**Comment by ameaninglessname** [comment]
Because `ALS-Refactored` is hosted on `GitHub`, generally you should use `git clone` to get it, and `git pull` to update it. A git GUI client like `TortoiseGit` would do better job for you.
